docs-src/dev-guide/sdk-features.md

```diff
@@ -0,0 +1,571 @@
+| ----------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Continue as New](#continue-as-new) | Yes | Yes  | Yes        | Yes    | Yes  | Yes |
+
+Continue-As-New is the mechanism by which all relevant state is passed to a new Workflow Execution with a fresh Event History.
+
+## Delay Start Workflow
+
+| Feature                                        | Go  | Java | TypeScript | Python | .Net | PHP |
+| ---------------------------------------------- | --- | ---- | ---------- | ------ | ---- | --- |
+| [Delay Start Workflow](#delay-start-workflow)) | Yes | Yes  | No         | No     | No   | No  |
+
+<!-- Track feature: https://github.com/temporalio/features/issues/338 -->
+
+A Delay Start Workflow is a Workflow that waits a specified duration before spawn the first Workflow.
+If the Workflow receives a Signal before the delay, the Workflow will be started immediately and the rest of the delay is ignored.
+
+A Start with Signal won't trigger the Workflow to start immediately and follow the delay.
+
+## Dynamic Entities
+
+Temporal supports Dynamic Workflows, Activities, Signals, and Queries.
+
+These are unnamed entities that are invoked if no other statically defined entity with the given name exists.
+
+| Feature                                   | Go | Java | TypeScript | Python | .Net | PHP |
+| ----------------------------------------- | -- | ---- | ---------- | ------ | ---- | --- |
+| [Dynamic Activities](#dynamic-activities) | No | Yes  | No         | Yes    | Yes  | No  |
+| [Dynamic Queries](#dynamic-queries)       | No | Yes  | No         | Yes    | Yes  | No  |
+| [Dynamic Signals](#dynamic-signals)       | No | Yes  | Yes        | Yes    | Yes  | No  |
+| [Dynamic Workflows](#dynamic-workflows)   | No | Yes  | Yes        | Yes    | Yes  | No  |
+
+Dynamic Entities should be used judiciously as a fallback mechanism rather than the primary approach.
+Overusing them can lead to maintainability and debugging issues down the line.
+
+## Enhanced stack trace
+
+| Feature                                       | Go | Java | TypeScript | Python | .Net | PHP |
+| --------------------------------------------- | -- | ---- | ---------- | ------ | ---- | --- |
+| [Enhanced stack trace](#enhanced-stack-trace) | No | No   | Yes        | No     | No   | No  |
+
+Enhanced stack trace in Temporal is a feature that allows you to capture and view detailed information about the execution of a Workflow.
+It provides a stack trace of all the threads owned by the Workflow Execution, which can be useful for troubleshooting issues in production.
+The `__stack_trace` query is a predefined query available in many SDKs that returns the stack trace.
+You can use the `tctl Workflow stack` command to query the stack trace of a Workflow Execution.
+This feature is especially helpful for identifying errors and blocks in a Workflow Definition.
+
+## gRPC interceptors
```

Comment:
Super advanced feature, does not deserve general call out. If we are listing features at this level, there are many many more we should list first.